src/Views/Staff.jsx

```diff
@@ -0,0 +1,67 @@
+import data from '../data/staff.json';
+import blankPfp from '../assets/img/discord-blank2.png';
+
+export const Staff = () => {
+    return (
+       <main className="has-text-centered title">
+           <h1 className="is-size-1 pagetitle">Codecademy Community Server Staff</h1>
+           <p className="is-size-4 mb-3">Meet our server staff!</p>
+           <section className="container has-background-white mb-6 p-3">
+               <h2 className="is-size-2">Admins</h2>
+               <p>Admins are the heads of our staff and the people who manage how the group is run.</p>
+               <div className="staff-list">
+                   {
+                       data["superadmins"].map((user) => {
+                            return <div className="staff-card has-background-white">
+                                <img className="profile-picture" src={user[1]}/>
+                                <p className="name super-admin">{user[0]}</p>
+                            </div>
+                       })
+                    }{
+                       data["admins"].map((user) => {
+                        return <div className="staff-card has-background-white">
+                                <img className="profile-picture" src={user[1]}/>
+                                <p className="name admin">{user[0]}</p>
+                            </div>
+                        })
+                    }
+
+               </div>
+           </section>
+           <section className="container has-background-white mb-6 p-3">
+               <h2 className="is-size-2">Moderators</h2>
+               <p>Moderators comprise our main staff team and aim to keep the peace and happiness in our community.</p>
+               <div className="staff-list">
+                    {
+                       data["mods"].map((user) => {
+                        return <div className="staff-card has-background-white">
+                                <img className="profile-picture" src={user[1]}/>
+                                <p className="name mod">{user[0]}</p>
+                            </div>
+                        })
+                    }
+                    <div className="staff-card has-background-white">
```

Comment:
Due to tgrtim's profile picture being stored locally (thanks to weirdness with how Discord themselves store blank pfps) it cannot be handled the normal way with a URL so must be done separately by manually importing the image.
This isn't great but is fine for now, although is something we probably should find a better work around for in future